.eslintrc

```diff
@@ -0,0 +1,46 @@
+{
+  "extends": ["eslint:recommended"],
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "amd": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 3
+  },
+  "globals": {
+    "spyOnEvent": false,
+    "JSON": true
+  },
+  rules: {
+    "camelcase": 1,
```

Comment:
shall be fine to set to 2 (error) rather than 1 (warning). if we want grunt to keep running regardless error, `--force` is the trick.